dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorCode.java

```diff
@@ -145,11 +145,14 @@
     E4000( "Relationship: `{0}` cannot link to itself" ),
     E4001( "Property `{0}` can not be set when property `{1}` is `{2}`. " ),
     E4002( "Property `{0}` must be set when property `{1}` is `{2}`. " ),
-    E4003( "There are duplicated relationships. " ),
-    E4004( "Missing required property: 'relationshipType'." ),
+    E4003( "There are duplicated relationships." ),
+    E4004( "Missing required relationship property: 'relationshipType'." ),
     E4005( "RelationShip: `{0}`, do not exist." ),
     E4006( "Could not find RelationShipType: `{0}`."  ),
```

Comment:
Could not find RelationShipType: `{0}`."  -> can you change to `relationshipType`?

Reply:
🚢

Reply:
:) gone